dashboard/config/application.rb

```diff
@@ -80,11 +80,14 @@ class Application < Rails::Application
     cache_bust_path = Rails.root.join('.cache_bust')
     ::CACHE_BUST = File.read(cache_bust_path).strip.gsub('.', '_') rescue ''
 
+    config.assets.paths << Rails.root.join('./public/blockly')
```

Comment:
Note: We add /public/blockly as a new asset path, which is a symlink that changes depending on the `use_my_apps` option, so in development you can still use your own apps build.